text/survey_analysis_report2.md

```diff
@@ -89,75 +92,62 @@ More generally, they were questioning the cost over benefit ratio of changing th
-We therefore propose to mirror these files both in the project folder and in a different folder.
-This is technically possible using the git submodule technology: the data is only once on the server, but can be provided twice on the computer. 
-Modifying one or the other version of the file will be synchronised using git technologies.
+We propose to keep data and code in different first level folder, and to create several new folders (in the different first level folders), for each new experiment. This could be easily automatised in your computer language of choice, and we are working on bringing this functionality to gin-tonic (see below). In addition some of these new folders could also have their own templates. For instance different data folder following a specific template could be added depending on the kind of experiment is done.
 
-On the other hand,
-the distinction between data and code is primordial for an efficient work and data management. 
-Indeed, data requires a different type of version control than code,
-it is therefore preferable to keep these different file types in different folders.
-In addition, code should be reusable on different datasets,
-it makes therefore sense to be able to share it independently of the data.
-And finally, it is easy to make code read data present in a different folder and there is no technical reason to keep data and code together.
-We will therefore keep an organization where the first level separate data and code.
-However, we want to implement a different solution that creates new folders in the different first level folders, for each new experiment. 
 
-# Gin-tonic solution
+## The project and cross-project level
 
-The gin tonic application (thats brings tonic vigor to the g-node infrastructure gin) is still in development, 
-but will implement the different steps we presented above.
-It relies on the git, git annex, and git submodule technologies. 
+We propose to mirror some folders (for shared figures, report and conference, and manuscripts) both in the project folder and in a different folder (fig. 4). In addition, we would propose to add a `labcommons` folder that could be shared and synchronised between all lab members using cloud technologies. This will be quite difficult to set manually (using alias folders for example), but using git submodule technology in gin tonic, it can be automated (see below).
+
+![Fig.4: Folder structure developed after analysing the survey. The purple point and arrows indicates that these folders are mirrored in different folder structures (using the gi submodule technology). One can for instance access (and update) the same files looking in the project manuscript folder, or by navigating to the manuscripts folder, and then to the project. For each experiments, additional folders will be created automatically in different folders.](../50_figures/pulication_ready_figures/fig4.png)
+
+# Gin-tonic implementation
+
```

Comment:
# Automated project management: Gin-tonic

The major challenge in sharing a common project structure is about maintaining the individual project repositories to be well-formed. As the project develops, the common structure can become easily broken by researchers adding _ad hoc_ temporally folders, without knowing where the files should belong to. It takes lots of time and effort, from both researchers and data curators alike, to remember and follow the rules in individual specific cases of adding files and folders.

To alleviate the cost of maintaining the project structure, here we propose an automated way of managing the development of project.